extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -179,6 +179,7 @@ export class AmpPanZoom extends AMP.BaseElement {
     this.element.classList.add('i-amphtml-pan-zoom');
     this.content_ = children[0];
     this.content_.classList.add('i-amphtml-pan-zoom-child');
+    this.setAsOwner(this.content_);
```

Comment:
Ownership requires you to do `pauseCallback`, `resumeCallback`, `setInViewport` and a few others.

Reply:
Can I call `scheduleLayout` without `setAsOwner`? No, right?

Reply:
You can, I don't think amp-pan-zoom should take ownership, Just blindly calling  `scheduleLayout` should work.  `amp-image-slider` had the same issue and fixed it this way: https://github.com/ampproject/amphtml/blob/master/extensions/amp-image-slider/0.1/amp-image-slider.js#L669

Reply:
Yup it does work. Is it behavior we can rely on (i.e. is `scheduleLayout` meant to be used without `setAsOwner`?

Reply:
yes, in this case it is a bit of a hack though. Most of the time it would do nothing. Legit use of it is `lightbox` (normal one, not scrollable version), it doesn't need to take ownership of its children but it also knows runtime has no laid out its children because it has been hidden, so when it opens it calls `scheduleLayout` to kick off resource manager.

AFAIK, the whole concept of ownership is going away with layers, is that right Justin?

Reply:
Correct, layers will eventually do away with ownership.